topics/ecology/tutorials/QGIS_Web_Feature_Services/tutorial.md

```diff
@@ -0,0 +1,136 @@
+
+In the Geographical Information System landscape, there is existing standards to help users deal with remote data. Most common webservices are Web Map Services (WMS) and Web Feature Services (WFS). If WMS allows users only to access and display maps stored remotely, WFS is giving access to the features of data so you can modify it and create your own data and maps.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Managing QGIS Galaxy interactive tool
+QGIS is now integrated in Galaxy as an interactive tool. This kind of tools is working differently than classical tools as it allows the user to interact with a dedicated graphical interface. This kind of tools is used to give access to Jupyter notebooks, RStudio or R Shiny apps for example. 
+
+To use QGIS, you need to use the {% tool [dedicated form](https://ecology.usegalaxy.eu/root?tool_id=interactive_tool_qgis) %}, you can specify input datasets from your hisrtory you want to use in QGIS, or not ;), then press the execute button to launch a QGIS instance. When the graphical user interface of QGIS is ready to be used, a URL will be displayed at the top of the Galaxy center panel. If you don't see it, you can see and access it through the "Active InteractiveTools" space of the "User" menu or you can click on {% icon galaxy-eye %} on the tool in the history.
+
+Once you finished your work on QGIS, if you want to retrieve data and/or entire project, you need to save files in /output, then quit QGIS properly through the "Project" Menu tab.
+
+> <hands-on-title>Deploy your own QGIS instance</hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Deploy a {% tool [QGIS instance](https://ecology.usegalaxy.eu/root?tool_id=interactive_tool_qgis) %}
+> 3. Access QGIS
+> 4. Save your project in /working/output folder
+>
+{: .hands_on}
+
+# Web Feature Services
+## Loading WFS layer
+
+> <hands-on-title>Loading WFS layer</hands-on-title>
+>
+> 1. Go to the top left of QGIS and click on "Layer"
+> 2. Click on the 1st option "Data Source Manager" 
+> 3. Select "WFS / OGC API - Features" on left part of the pop-up panel
+> 4. Click the "New" button
+> 5. In the dialog that appears, enter the Name as `nsidc` and the URL as `http://nsidc.org/cgi-bin/atlas_south?version=1.1.0`.
+>
+>    > <tip-title>copy pasting from computer to QGIS</tip-title>
+>    >
+>    > * You can expand the QGIS left panel (where there is 3 dots, verticaly) to access the "clipboard" menu, and paste the content you want to paste on a QGIS form. Then, click outside of this panel to collapse it, and you can click for example on the `url` field to paste the url from your clipbaord
+>    >
+>    > ![QGIS clipboard](../../images/QGIS/qgistuto1.PNG)
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please consider checking all of them.*